medusa/server/web/home/add_shows.py

```diff
@@ -308,7 +310,7 @@ def popularShows(self):
 
         try:
             recommended_shows = ImdbPopular().fetch_popular_shows()
-        except (StandardError, requests.RequestException) as e:
+        except (RequestException, socket.gaierror, StandardError) as e:
```

Comment:
Show we the error where it's raised by requests? I'm not interested if RequestsExceptions handles it or not. I know it won't handle that. I'm interested if it can be raised by a request.session.get()..